### PR TITLE
Create author class

### LINKED ISSUE #27: [1.5pt] Create Author class
- Create an Author class with an association to the Item class (in a separate .rb file).
- All Author class properties visible in the diagram should be defined and set up in the constructor method.

### PR DESCRIPTION
**Pull request by SelmaNdapanda**
### In this pull request, I implemented the following:  :arrow_down: 

- Create an Author class with an association to the Item class (in a separate .rb file).  :heavy_check_mark: 
-  All Author class properties visible in the diagram should be defined and set up in the constructor method.  :heavy_check_mark:

### CODE REVIEW

File: author.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class Author
+  attr_accessor :id, :first_name, :last_name, :items
+
+  def initialize(first_name, last_name)
+    @id = Random.rand(1..1000)
+    @first_name = first_name
+    @last_name = last_name
+    @items []
```
Comment (blase147) [inline author.rb]: The instance variable for items isn't set to the empty array yet. I suggest you add '=' to fix this oversight.

**Review by blase147 — COMMENTED** [review]
## Changes Required♻
Hi @SelmaNdapanda Great job 👍 at implementing the required codes but there is an issue you must fix before this project is approved 

## Highlights
Linters are passing
Good GitHub flow

## Status: ✔️
 -  Kindly check the inline comments under the review.

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

File: author.rb (hunk 1 above)
Comment (SelmaNdapanda) [reply author.rb]: Okay, thank you. I will sort it out now

**Review by myaserkhan — APPROVED** [review]
## Hi :raising_hand:  @SelmaNdapanda, 

Great work on making the changes requested by a previous reviewer 👏

## Highlights :green_circle: 

- All Linter checks passed :+1: 

Cheers and Happy coding!👏👏👏

**Pull request merged**